src/main/java/org/reaktivity/nukleus/tls/internal/stream/ClientStreamFactory.java

```diff
@@ -240,6 +240,7 @@ private RouteFW wrapRoute(
         private int applicationWindowBytesAdjustment;
         private int applicationWindowFrames;
         private int applicationWindowFramesAdjustment;
+        private boolean applicationEnded;
```

Comment:
Suggest setting `applicationWindowBytes` to `-1` instead of adding the extra `applicationEnded` field.